core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -281,57 +340,14 @@ oppia.directive('conversationSkin', [function() {
             $scope.activeCard);
         };
 
-        var _recomputeAndResetPanels = function() {
-          $scope.clearHelpCard();
-
-          $scope.panels = [];
-          if (!$scope.canWindowFitTwoCards()) {
-            $scope.panels.push($scope.PANEL_TUTOR);
-          }
-          if ($scope.isCurrentSupplementalCardNonempty()) {
-            $scope.panels.push($scope.PANEL_SUPPLEMENTAL);
-          }
-          $scope.resetVisiblePanel();
-        };
-
-        $scope.currentVisiblePanelName = null;
-
-        $scope.isPanelVisible = function(panelName) {
-          if (panelName === $scope.PANEL_TUTOR &&
-              $scope.canWindowFitTwoCards()) {
-            return true;
-          } else {
-            return panelName == $scope.currentVisiblePanelName;
-          }
-        };
-
-        $scope.setVisiblePanel = function(panelName) {
-          $scope.currentVisiblePanelName = panelName;
-          if (panelName === $scope.PANEL_TUTOR) {
-            $scope.clearHelpCard();
-          }
-          if (panelName === $scope.PANEL_SUPPLEMENTAL) {
-            $scope.$broadcast('showInteraction');
-          }
-        };
-
-        $scope.resetVisiblePanel = function() {
-          if ($scope.panels.length === 0) {
-            $scope.currentVisiblePanelName = null;
-          } else {
-            $scope.currentVisiblePanelName = $scope.panels[0];
-          }
-        };
-
         // Navigates to the currently-active card, and resets the 'show previous
         // responses' setting.
         var _navigateToActiveCard = function() {
           var index = playerPositionService.getActiveCardIndex();
           $scope.activeCard = playerTranscriptService.getCard(index);
           $scope.arePreviousResponsesShown = false;
           $scope.clearHelpCard();
-
-          _recomputeAndResetPanels();
+          $scope._tutorCardDisplayedOnNarrow = true;
```

Comment:
Drop leading underscore (we're moving away from that convention, see [style rules](https://github.com/oppia/oppia/blob/develop/CONTRIBUTING.md#style-rules)). Use var instead of $scope to indicate that this is local to this controller.

Also, please add initialization of this variable at the top (around line 285 or so) and a one-line comment explaining what it means, as precisely as possible. (Is it actually supposed to be tutorCardDisplayedIfNarrow? I'm not totally sure whether this means "display the tutor card once a screen-has-just-become-narrow event is received", or "display the tutor card whenever the screen is narrow".)